augraphy/augmentations/dustyink.py

```diff
@@ -36,7 +47,8 @@ def __repr__(self):
     def __call__(self, image, layer=None, force=False):
         if force or self.should_run():
             image = image.copy()
-            apply_mask_fn = lambda x, y: y if (x < 64) else x
```

Comment:
I added `value_range` so that user is able to choose the min value of pixel to enable the dusty ink effect. Previously it was set at 64, which might not work all the time.